recipes-samples/images/rpb-minimal-image.bb

```diff
@@ -12,3 +12,7 @@ REQUIRED_DISTRO_FEATURES = "pam systemd"
 CORE_IMAGE_BASE_INSTALL += " \
     kernel-modules \
 "
+
```

Comment:
same here.

Reply:
I will remove the new line.

Reply:
Oh i got it what you said, i don't know why github only show a diff of this line and yes i will make the same here to integrate into separate commit.